packages/blanket/src/Blanket.tsx

```diff
@@ -9,6 +9,7 @@ export const PureBlanket: React.FC<IBlanketProps> = ({
   theme,
   canClickThrough = false,
   isTinted = false,
+  maxZIndex,
```

Comment:
Blanket and Modal don't need to know about maxZIndex. They should only accept zIndex.